evm/src/all_stark.rs

```diff
@@ -87,40 +86,74 @@ mod tests {
         let keccak_stark = KeccakStark::<F, D> {
             f: Default::default(),
         };
-        let keccak_rows = (2 * NUM_ROUNDS + 1).next_power_of_two();
-        let keccak_looked_col = 3;
 
-        let mut rng = ChaCha8Rng::seed_from_u64(0x6feb51b7ec230f25);
+        let mut rng = thread_rng();
         let num_inputs = 2;
```

Comment:
Can we call this something like `num_keccak_perms`? A bit confusing with this and `NUM_INPUTS`